### PR TITLE
Upgrade ain-util package version to v1.2.1

### PR DESCRIPTION
**Pull request by platfowner**
Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1282

### CODE REVIEW

File: tools/transfer/sendTransferTx.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ function buildTransferTxBody(fromAddr, toAddr, key, amount, timestamp) {
     gas_price: 500,
     timestamp,
     nonce: -1
+//    nonce: -2
+//    nonce: 0
```
Comment (jiyoung-an) [inline tools/transfer/sendTransferTx.js:22]: Is it intentional code?

**Review by jiyoung-an — APPROVED** [review]
LGTM, Thanks!

File: tools/transfer/sendTransferTx.js (hunk 1 above)
Comment (platfowner) [reply tools/transfer/sendTransferTx.js]: yes, just added a note.

**Comment by platfowner** [comment]
Thanks for the review! @jiyoung-an Now merging..

**Pull request merged**